homework Array.playground/Contents.swift

```diff
@@ -0,0 +1,55 @@
+var myArray = [1,10,22,44,5,13,19,23,99,9]
+
+print("Orignal Array = \(myArray)")
+
+print("The reversed Array = ")
+
+for item in myArray.reversed()
+{print(item)}
+
+print("The Random Element of Array = \(myArray.randomElement()!)")
```

Comment:
@AGAHZWNI great job but this will give us a random element in the array
but we want to sort the array in a random way for example [1,2,3] =>[3,1,2]